src/plugins/intel_gpu/src/graph/impls/ocl/kernel_selector_helper.h

```diff
@@ -275,4 +275,23 @@ inline std::shared_ptr<WeightsReorderParams> create_weights_reorder_params(const
     return std::make_shared<WeightsReorderParams>(from_weights_tensor(params.src), from_weights_tensor(params.dest), params.rotate);
 }
 
+inline void update_shapes(kernel_selector::Params& p, const kernel_impl_params impl_param) {
```

Comment:
Can we pass `impl_param` here by reference too?

Reply:
good catch. updated